images/build/go-runner/Makefile

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# set default shell
+SHELL=/bin/bash -o pipefail
+
+REGISTRY ?= gcr.io/k8s-staging-build-image
+IMGNAME = go-runner
+IMAGE = $(REGISTRY)/$(IMGNAME)
+
+TAG ?= $(shell git describe --tags --always --dirty)
+IMAGE_VERSION ?= buster-v1.0.0
+CONFIG ?= buster
```

Comment:
why are we using `buster`? this is distroless right?

Reply:
digging deeper with links from @justaugustus 
https://github.com/GoogleContainerTools/distroless#docker
https://github.com/GoogleContainerTools/distroless#base-operating-system

LGTM

Reply:
(Was already chatting with Dims on this, but calling it out here as well...)

While distroless images are stripped-down, they still have a Debian base.
Previously, `distroless/static` and `distroless/base` images were built on Debian 9 (Stretch).

Here, we're just calling out that we explicitly that we want to use the Debian 10 (Buster) variants of these images instead.

ref: https://github.com/GoogleContainerTools/distroless#base-operating-system, https://github.com/GoogleContainerTools/distroless#docker